templates/index.html

```diff
@@ -385,6 +394,12 @@ <h5 class="type"></h5>
             ga('create', 'UA-48606400-1', 'anyway.herokuapp.com');
             ga('send', 'pageview');
         </script>
+	
+	<script type="text/javascript" src="../static/js/tutorial.js"> </script>
```

Comment:
I tried but got an webasset error. I moved all the css to the assets. The javascript files are already minified.